scripts/linters/pre_commit_linter.py

```diff
@@ -314,33 +312,33 @@ def _get_all_files_in_directory(dir_path, excluded_glob_patterns):
     return files_in_directory
 
 
-def _get_file_extensions(file_extension_type):
+def _get_file_extensions(file_extensions_to_lint):
     """This function is used to return the file extensions which need to be
     linted and checked.
 
     Args:
-        file_extension_type: list(str). The list of file extensions to be
+        file_extensions_to_lint: list(str). The list of file extensions to be
         linted checked.
 
     Returns:
         all_file_extensions_type: list(str). The list of all file extensions
         to be linted and checked.
     """
-    all_file_extensions_type = ['*.js', '*.py', '*.html', '*.css']
+    all_file_extensions_type = ['js', 'py', 'html', 'css', 'other']
 
-    if file_extension_type:
-        # Check if '*.js' and '*.ts' both are present in file_extension_type.
-        js_and_ts_is_present = '*.js' in file_extension_type and (
-            '*.ts' in file_extension_type)
+    if file_extensions_to_lint:
+        # Check if 'js' and 'ts' both are present in file_extensions_to_lint.
+        js_and_ts_is_present = 'js' in file_extensions_to_lint and (
+            'ts' in file_extensions_to_lint)
 
         if js_and_ts_is_present:
             python_utils.PRINT(
-                'Please either use "*.js" or "*.ts" both are not '
+                'Please either use "js" or "ts". Both are not '
```

Comment:
If you're keeping this message, explain why, so that the developer has context for what's going on behind the scenes.

Reply:
Done.